modules/kafka/src/main/scala/higherkindness/mu/rpc/kafka/kafkaManagementService.scala

```diff
@@ -291,43 +316,46 @@ object kafkaManagementService {
         if (koam.leaderEpoch().isPresent()) Some(koam.leaderEpoch().get) else None
       )
   }
-  final case class Offset(topicPartition: TopicPartition, metadata: OffsetAndMetadata)
-  final case class ConsumerGroupOffsets(offsets: List[Offset])
+  final case class Offset(
+      @pbIndex(1) topicPartition: TopicPartition,
+      @pbIndex(1) metadata: OffsetAndMetadata)
+  final case class ConsumerGroupOffsets(@pbIndex(1) offsets: List[Offset])
 
   final case class ConsumerGroupListing(
-      groupId: String,
-      isSimpleConsumerGroup: Boolean
+      @pbIndex(1) groupId: String,
+      @pbIndex(2) isSimpleConsumerGroup: Boolean
   )
   object ConsumerGroupListing {
     def fromJava(kcgl: KConsumerGroupListing): ConsumerGroupListing = ConsumerGroupListing(
       kcgl.groupId(),
       kcgl.isSimpleConsumerGroup()
     )
   }
-  final case class ConsumerGroupListings(consumerGroupListings: List[ConsumerGroupListing])
+  final case class ConsumerGroupListings(
+      @pbIndex(1) consumerGroupListings: List[ConsumerGroupListing])
 
   final case class TopicListing(
-      name: String,
-      isInternal: Boolean
+      @pbIndex(1) name: String,
+      @pbIndex(2) isInternal: Boolean
   )
   object TopicListing {
     def fromJava(ktl: KTopicListing): TopicListing = TopicListing(ktl.name(), ktl.isInternal())
   }
-  final case class TopicListings(listings: List[TopicListing])
+  final case class TopicListings(@pbIndex(1) listings: List[TopicListing])
 
   @service(Protobuf)
   trait KafkaManagement[F[_]] {
     def alterConfigs(acr: AlterConfigsRequest): F[Unit]
     def createPartitions(cpr: CreatePartitionsRequest): F[Unit]
     def createTopic(ctr: CreateTopicRequest): F[Unit]
     def createTopics(ctrs: CreateTopicRequests): F[Unit]
-    def deleteTopic(t: String): F[Unit]
```

Comment:
Here's an example of an RPC request that's no longer allowed

Reply:
Yeah, it makes sense if we need to wrap it up into a case class